lib/dev.js

```diff
@@ -107,7 +107,10 @@ module.exports = async function dev (sourceDir, cliOptions = {}) {
     compiler,
     host,
     dev: { logLevel: 'warn' },
-    hot: { logLevel: 'error' },
+    hot: {
+      port: parseInt(port) + 1,
```

Comment:
`portfinder.getPortPromise` will return `number`, so `parseInt ` is unnecessary here.